IntelliJ-config/saveactions_settings.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
let's have a config directory for this, so move this to the `config/ide/intellij/saveactions_settings.xml`.. We will surely have more configuration to come.. update the description in the contribute file..